hirosystems/stacks-blockchain-api/templates/api-writer/statefulset.yaml

```diff
@@ -145,9 +145,31 @@ spec:
               subPath: {{ .Values.apiWriter.persistence.data.subPath }}
               {{- end }}
         {{- end }}
-        {{- if .Values.apiWriter.config.dataArchiveUrl }}
+        {{- if and (.Values.apiWriter.config.exportEvents.enabled) (.Values.apiWriter.config.exportEvents.compressEventsData) }}
+        - name: compress-events-data
+          image: google/cloud-sdk:alpine
```

Comment:
Curious why you settled on this image as opposed to just `alpine`?

Reply:
this is the same image we're using in the api-archive env.

Reply:
ha I forgot, thanks for double checking!